### PR TITLE
Add ConfigCacheDocsTest to TeamCity

### PR DESCRIPTION
**Pull request by blindpirate**
Add a `docsTest` with `-PenableConfigurationCacheForDocsTests=true`, as requested by @mlopatkin .

### CODE REVIEW

File: .teamcity/src/main/kotlin/configurations/ConfigCacheDocsTest.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    applyTestDefaults(
+        model,
+        this,
+        "docs:docsTest",
+        timeout = 60,
+        extraParameters = buildScanTag("ConfigCacheDocsTest") +
+            " -PenableConfigurationCacheForDocsTests=true" +
+            " -PtestJavaVersion=${testJava.version.major}" +
+            " -PtestJavaVendor=${testJava.vendor.name}" +
+            " -Porg.gradle.java.installations.auto-download=false"
```
Comment (mlopatkin) [inline .teamcity/src/main/kotlin/configurations/ConfigCacheDocsTest.kt:30]: `auto-download=false` seems to come from `applyTestDefaults` too, maybe we could omit it here?

**Comment by blindpirate** [comment]
@bot-gradle test and merge

File: .teamcity/src/main/kotlin/configurations/ConfigCacheDocsTest.kt (hunk 1 above)
Comment (blindpirate) [reply .teamcity/src/main/kotlin/configurations/ConfigCacheDocsTest.kt]: You are right. Fixed.

**Review by mlopatkin — APPROVED** [review]
LGTM, thank you very much!

**Pull request merged**